### PR TITLE
expose the k8s api version and groups as params

### PR DESCRIPTION
**Pull request by vsinghal13**
###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/kubernetes/fluentd-sumologic.yaml.tmpl [hunk 1]
```diff
@@ -85,8 +85,8 @@ data:
         cache_size  "10000"
         cache_ttl  "3600"
         cache_refresh "1800"
-        core_api_versions [v1]
-        api_groups [apps/v1 extensions/v1beta1]
+        core_api_versions "[v1]"
+        api_groups "[apps/v1 extensions/v1beta1]"
```
Comment (vsinghal13) [inline deploy/kubernetes/fluentd-sumologic.yaml.tmpl:89]: @samjsong do we need quotes? Also, inside the brackets, the two items are with no quotes and  no comma

Comment (vsinghal13) [reply deploy/kubernetes/fluentd-sumologic.yaml.tmpl:89]: cc @perk-sumo

Comment (samjsong) [reply deploy/kubernetes/fluentd-sumologic.yaml.tmpl:89]: that's interesting, can you try a non-helm install and see if this works?

**Comment by vsinghal13** [comment]
Converted to comma-separated list based on : https://docs.fluentd.org/configuration/config-file
```
array type: the field is parsed as a JSON array. It also supports
shorthand syntax. These are same values.
normal: ["key1", "key2"]
shorthand: key1,key2
```

**Comment by vsinghal13** [comment]
> LGTM :) but the `Non_Helm_Installation.md` doc changes seem out of place, is that a result of some merge from master or something?

fixed it

**Pull request merged**